tests/src/model/pages/containers-page.ts

```diff
@@ -17,39 +17,27 @@
  ***********************************************************************/
 
 import type { Locator, Page } from 'playwright';
-import { PodmanDesktopPage } from './base-page';
+import { MainPage } from './main-page';
 import { ContainerDetailsPage } from './container-details-page';
 
-export class ContainersPage extends PodmanDesktopPage {
-  readonly heading: Locator;
+export class ContainersPage extends MainPage {
   readonly pruneContainersButton: Locator;
   readonly createContainerButton: Locator;
   readonly playKubernetesYAMLButton: Locator;
+  readonly deleteSelectedButton: Locator;
+  readonly createPodButton: Locator;
```

Comment:
What are they needed for? I just see them in the constructor

Reply:
They will be added as a part of another PRs where they will be actually used (and tested). But I guess I should remove them and put a todo comment instead?

Reply:
You could just remove them and add them back together with the new PR. It makes mroe sense to me.